tests/FSharp.Core.UnitTests/FSharp.Core/Microsoft.FSharp.Control/AsyncModule.fs

```diff
@@ -469,21 +469,27 @@ type AsyncModule() =
     member _.``error on one workflow should cancel all others``() =
         task {
             use failOnlyOne = new Semaphore(0, 1)
-            let mutable cancelled = 0
-            let mutable started = 0
+            // Start from 1.
+            let mutable running = new CountdownEvent(1)
```

Comment:
CountdownEvent to be sure all cancellations are handled.

Reply:
https://dev.azure.com/dnceng-public/public/_build/results?buildId=854431&view=logs&j=09cc464c-89cd-5758-006d-ba9e3958cfa9&t=098678ff-b2f3-5c88-44d3-4c7be2f47e75&l=16973 oh well, this is tricky to get right.